sensehat-core.c

```diff
@@ -115,35 +120,22 @@ sensehat_client_dev_register(struct sensehat *sensehat, const char *name)
 	return ERR_PTR(ret);
 }
 
-int sensehat_get_joystick_state(struct sensehat *sensehat)
+static bool sensehat_writeable_register(struct device *dev, unsigned reg)
 {
-	int ret = i2c_smbus_read_byte_data(sensehat->i2c_client, RPISENSE_KEYS);
-
-	return ret < 0 ? ret : ret & 0x1f;
+	return (0<=reg && reg<=191) || reg==243;
```

Comment:
definitely #define these magic numbers

Reply:
also these numbers make more sense in hex I think since the bit boundaries are cleraer